.github/workflows/e2e-infra-cleanup.yaml

```diff
@@ -0,0 +1,78 @@
+name: e2e-infra-cleanup
+
+on:
+  workflow_dispatch:
+    inputs:
+      cleanup_interval_hours:
+        description: Cleanup interval in hours
+        required: true
+        default: "24"
+    secrets:
+      E2E_TESTIM_AWS_ACCESS_KEY_ID:
+        required: true
+      E2E_TESTIM_AWS_SECRET_ACCESS_KEY:
+        required: true
+      E2E_GH_PAT:
+        required: true
+  schedule:
+    - cron: "0 * * * *"
+
+concurrency: e2e-infra-cleanup
+
+env:
+  AWS_DEFAULT_REGION: us-east-1
+  AWS_ACCESS_KEY_ID: ${{ secrets.E2E_TESTIM_AWS_ACCESS_KEY_ID }}
+  AWS_SECRET_ACCESS_KEY: ${{ secrets.E2E_TESTIM_AWS_SECRET_ACCESS_KEY }}
```

Comment:
Are these populated for cron?

Reply:
The secrets context should be available for all runs. Unfortunately not a way to test it without being on the main branch. We'll need to monitor the automated cleanup and address if it's failing.